addons/website_livechat/static/src/components/visitor_banner/visitor_banner.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="website_livechat.VisitorBanner" owl="1">
-        <div class="o_VisitorBanner">
+        <div class="o_VisitorBanner border-bottom">
```

Comment:
`border-bottom` is contextual style to `ThreadView`, so it shouldn't be in `VisitorBanner` by itself.
Prefer passing class in `ThreadView` to `VisitorBanner`:

```js
<VisitorBanner class="border-bottom"/>
```

During forward-port, at 15.2, make sure to replace above LOC with this:
```js
<VisitorBanner className="'border-bottom'"/>
```

_(FYI, passing `class` only works in OWL v1 and 15.2 did code prepare for OWL v2 that is currently used in master)_